projects/ngrx.io/content/guide/signals/signal-store/index.md

```diff
@@ -217,7 +217,7 @@ export const BooksStore = signalStore(
 
 <div class="alert is-helpful">
 
-The state of the SignalStore is updated using the `patchState` function.
+The state of the SignalStore is frozen in development mode, which means any mutable change will throw an error. `patchState` is the function to update the state.
 For more details on the `patchState` function, refer to the [Updating State](/guide/signals/signal-state#updating-state) guide.
```

Comment:
Since we're already referencing the 'Updating State' guide in the alert, we can keep this alert as it is to avoid duplication.